lib/msal-node/docs/caching.md

```diff
@@ -0,0 +1,115 @@
+# Token caching in MSAL Node
+
+When MSAL Node acquires a token, it caches it in memory for future usage. MSAL Node manages the token lifetime and refreshing for you. APIs like `acquireTokenSilent()` retrieves the access tokens from the cache for a given account:
+
+```javascript
+const msal = require('@azure/msal-node');
+
+// Create msal application object
+const cca = new msal.ConfidentialClientApplication({
+    auth: {
+        clientId: "Enter_the_Application_Id_Here",
+        authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here",
+        clientSecret: "Enter_the_Client_Secret_Here"
+    }
+});
+
+// login* and acquireToken* APIs return an account object containing "homeAccountId"
+// you should keep a record of this in your app and use it later on when calling acquireTokenSilent
+const someUserHomeAccountId = "Enter_User_Home_Account_Id";
+
+const msalTokenCache = cca.getTokenCache();
+const account = await msalTokenCache.getAccountByHomeId(someUserHomeAccountId)
+
+const silentTokenRequest = {
+    account: account,
+    scopes: ["user.read"],
+};
+
+cca.acquireTokenSilent(silentTokenRequest).then((response) => {
+    // do something with response
+}).catch((error) => {
+    // catch and handle errors
+});
+```
+
+MSAL's in-memory token cache is not suitable for production. In production, you should serialize and persist the token cache. Depending on the type of application, you have several alternatives:
```

Comment:
We should call out encryption too with persistence.

Reply:
added 👍